frontend/src/scenes/App.tsx

```diff
@@ -129,6 +130,7 @@ function AppScene(): JSX.Element | null {
             </Navigation>
             {toastContainer}
             <UpgradeModal />
+            <DefinitionPanel />
```

Comment:
Wonder why we need to mount this at such high-level, perhaps we mount it together with the TaxonomicFilter? That's where we'll use it anyways.

Reply:
👍 I'll push it down to the taxonomic filter level. There're a few more places we'll want to put this drawer but I think it makes sense to add it in those contexts separately